plugins/services/src/js/components/modals/ServiceActionDisabledModal.js

```diff
@@ -363,7 +363,7 @@ class ServiceActionDisabledModal extends React.Component {
   getGroupDeleteMessage() {
     const { actionID, service, intl } = this.props;
     const serviceID = service ? service.getId() : "";
-    const { selectedServices } = this.getServiceTypes(isPackage);
+    const { selectedServices } = this.getServiceTypes(isSDKService);
```

Comment:
@MatApple here is some more information, hope it makes sense:
The user action here is to delete a group. In the group where we want to delete services installed from Cosmos (Packages) and apps (Marathon) differently. Packages installed from Cosmos needs to be deleted using the `dcos package uninstall ${packageName} --app-id=${serviceID}`-command and run the cleaner script. Only applications installed directly through Marathon can be removed with the group delete command; `dcos marathon group remove ${groupID}`.

This deviates slightly from what we are doing in general since we just run the Marathon delete command on everything (which we should change), but this is the right thing to do.